### PR TITLE
Graph-bulk-meetings node js sample issues fixed and image updated

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**

### CODE REVIEW

File: samples/graph-bulk-meetings/nodejs/controller/meeting-controller.js [hunk 1]
```diff
@@ -9,8 +9,10 @@ const createMeetingAsync = async (req, res) => {
     var excelData = req.body;
```
Comment (Pawank-MSFT) [inline samples/graph-bulk-meetings/nodejs/controller/meeting-controller.js:9]: from where createMeetingAsync calling?

**Review by Pawank-MSFT — APPROVED** [review]
please fix small comment, rest looks correct, Approving!

File: samples/graph-bulk-meetings/nodejs/controller/meeting-controller.js (hunk 1 above)
Comment (Harikrishnan-MSFT) [reply samples/graph-bulk-meetings/nodejs/controller/meeting-controller.js:9]: image

image
image

**Comment by Harikrishnan-MSFT** [comment]
Build Pipeline  - 
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/7911016100
image

**Review by Prithvi-MSFT — APPROVED** [review]

**Comment by Harikrishnan-MSFT** [comment]
**Build Pipeline -
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/8059993435**
image

**Pull request merged**